mvp-spec/workflows.md

```diff
@@ -0,0 +1,84 @@
+# Workflows
+
+The workflows part of the Prebid Addressability Framework are:
+- User Id and Preferences
+- Selling ad slots
+
+## Id and User Preferences
+
+### Generic workflow
+
+This workflow illustrates retrieving and creating/setting User Id and Preferences.
+
+```mermaid
+sequenceDiagram
+participant U as User
+participant P as Participant's website
+participant O as Operator
+
+    U ->> P: visit site
+    
+    P ->> O: Read User Id and Preferences
+    O ->> P: No User Id and Preferences
+    
+    P ->> U: Display consent prompt
+    U ->> P: Give consent
+
+    P ->> O: Write User Id and Preferences
+    O ->> O: Generate<br />User Id and Preferences
```

Comment:
I think this sentence is misleading: the preferences have already been generated (the Id as well, but in the case of the Id, it's an implementation detail).
At this stage, the Id and Preferences are **saved**, or **stored**.
Or maybe a better phrasing, in my opinion, could be "Generate User Id and Preferences PAF cookies".

Reply:
(apart from that, I think this high level diagram is great 😉)

Reply:
Ok, diagram updated in a new commit. I also added an "ack" sequence at the end of the diagram, please let me know if it doesn't work like that.